src/firstvoices/backend/views/base_views.py

```diff
@@ -14,9 +14,13 @@ class FVPermissionViewSetMixin(AutoPermissionViewSetMixin):
     model permissions are registered using ``rules.contrib.models.RulesModelMixin``.
     """
 
+    # Method to override the list queryset to implement different querysets for list and retrieve respectively
+    def get_list_view_queryset(self):
```

Comment:
Thanks for adding this. It's going to be so handy. We already have 2 cases where the list and detail views have different querysets!